ci/build_setup.sh

```diff
@@ -80,6 +80,7 @@ cleanup
 trap cleanup EXIT
 
 bazel/setup_clang.sh /opt/llvm
+export LLVM_CONFIG=/opt/llvm/bin/llvm-config
```

Comment:
I don't think you need this because setup_clang.sh does add this in action_env?

Reply:
Yeah, but the `LLVM_CONFIG` is needed at repository rule evaluation time to setup `@envoy_dev`, before `action_env` has any effect.

Reply:
Hmm, does [`--repo_env`](https://docs.bazel.build/versions/master/command-line-reference.html#flag--repo_env) works?

Reply:
Yes, but `api_booster.py` still relies directly on `LLVM_CONFIG` as well. I've switched to using `repo_env` for the basic Bazel setup, as this makes things a bit cleaner, but I think this is as clean as we can figure out how to get rid of https://github.com/envoyproxy/envoy/blob/674b82c1d4bce2cf49793ab975e1aeeb189abe58/tools/api_boost/api_boost.py#L132.